### ISSUE TITLE
Reduce scroll of pager function a little

### ISSUE THREAD

**Issue by Arzack** [pratikabu/scrolltotop#53]
Sometime the pager function happens to cut a line of text (or even worse, half a reddit post). I suggest a 270 pixel count scroll to solve the issue (or a slider option).

**Comment by pratikabu** [comment]
I agree with your post, the page up and page down calculations are actually little complex and varies screen to screen. Also, it depends on your browser window size.

I'll take a look at it after the Firefox Webextension release.
